src/Deal/DealListRequestOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Skarp.HubSpotClient.Deal
+{
+    public class DealListRequestOptions
+    {
+        private int _numberOfCompaniesToReturn = 100;
+
+        /// <summary>
+        /// Gets or sets the number of deals to return.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to 20 which is also the hubspot api default. Max value is 100
+        /// </remarks>
+        /// <value>
+        /// The number of deals to return.
+        /// </value>
+        public int NumberOfCompaniesToReturn
```

Comment:
shouldn't this be called `NumberOfDealsToReturn` since this class is a `DealListRequestOptions`?